packages/nx-aws-cache/src/tasks-runner/aws-cache.ts

```diff
@@ -2,39 +2,37 @@ import { createReadStream, createWriteStream, writeFile } from 'fs';
 import { join, dirname } from 'path';
 import { pipeline, Readable } from 'stream';
 import { promisify } from 'util';
-
 import * as clientS3 from '@aws-sdk/client-s3';
 import { fromNodeProviderChain } from '@aws-sdk/credential-providers';
 import { CredentialsProviderError } from '@aws-sdk/property-provider';
 import { RemoteCache } from '@nx/workspace/src/tasks-runner/default-tasks-runner';
 import { create, extract } from 'tar';
-
 import { AwsNxCacheOptions } from './models/aws-nx-cache-options.model';
 import { Logger } from './logger';
 import { MessageReporter } from './message-reporter';
+import { Encrypt, Decrypt, EncryptConfig } from './encryptor';
+import { Upload } from '@aws-sdk/lib-storage';
 
 export class AwsCache implements RemoteCache {
   private readonly bucket: string;
   private readonly path: string;
   private readonly s3: clientS3.S3Client;
   private readonly logger = new Logger();
   private readonly uploadQueue: Array<Promise<boolean>> = [];
+  private readonly encryptConfig: EncryptConfig | null = null;
 
   public constructor(options: AwsNxCacheOptions, private messages: MessageReporter) {
     const awsBucket = options.awsBucket ?? '';
     const bucketTokens = awsBucket.split('/');
     this.bucket = bucketTokens.shift() as string;
     this.path = bucketTokens.join('/');
+    this.encryptConfig = null;
 
     const clientConfig: clientS3.S3ClientConfig = {};
 
-    if (options.awsRegion) {
-      clientConfig.region = options.awsRegion;
-    }
+    if (options.awsRegion) clientConfig.region = options.awsRegion;
```

Comment:
Please revert brackets after if statements.

Reply:
Fixed, bu you have setup limit of 300 lines per file

Reply:
We can increase this limit to 400 or split this file into multiple meaningful files.